src/foam/u2/borders/CollapseBorder.js

```diff
@@ -91,20 +84,15 @@ foam.CLASS({
         addClass(this.myClass()).
         enableClass('expanded', this.expanded$).
         start('div').
+          on('click', this.toggle.bind(this)).
```

Comment:
Should use the action instead, onClick isnt picked up by screen readers or keyboard navigation